### DISCUSSION TITLE
How to do collision detection?

### DISCUSSION THREAD

**Discussion by Lasse-Fisker** [spritewidget/spritewidget#78] [Q&A]
Hi. First of all, thanks for the great work on the library! 

I was wondering how you would recommend doing collision detection? I'm thinking that there should be a relatively easy way to check if a node is touching other nodes, but I can't seem to figure it out.

Thank you in advance.

**Comment by erik-seifert** [comment]
Some examples with point conversion would be very nice.

**Comment by kevinvandenbreemen** [comment]
I was actually thinking of collision detection when I read the section on constraints (https://github.com/spritewidget/spritewidget#defining-constraints ).  Is that something that could be extended to accomplish collision detection?
